### PR TITLE
Update to Forge 41.0.98

### PR DESCRIPTION
**Pull request by zach2039**
Exactly what is says on the tin; mainly refactor for things still there, just under a different name or import.

_EDIT: Commit message is wrong; this is indeed for 41.0.98._

### CODE REVIEW

**Comment by MaxNeedsSnacks** [comment]
Thanks for the contribution! Forgot to subscribe to this repo at first so I missed it :P

File: src/main/java/dev/latvian/mods/tanky/client/TankControllerRenderer.java [hunk 1]
```diff
@@ -36,15 +34,15 @@ public void render(TankControllerBlockEntity entity, float delta, PoseStack matr
 		int light = LevelRenderer.getLightColor(entity.getLevel(), entity.tank.getFluid().getFluid().defaultFluidState().createLegacyBlock(), entity.getBlockPos().above());
 
 		FluidStack fluid = entity.tank.getFluid();
-		var renderProperties = RenderProperties.get(fluid.getFluid());
+		var renderProperties = ((IClientFluidTypeExtensions)fluid.getFluid().getFluidType().getRenderPropertiesInternal());
```
Comment (MaxNeedsSnacks) [inline src/main/java/dev/latvian/mods/tanky/client/TankControllerRenderer.java]: Couldn't this be `IClientFluidTypeExtensions.of` instead?

**Review by MaxNeedsSnacks — APPROVED** [review]

File: src/main/java/dev/latvian/mods/tanky/client/TankControllerRenderer.java (hunk 1 above)
Comment (zach2039) [reply src/main/java/dev/latvian/mods/tanky/client/TankControllerRenderer.java]: Didn't know it was an Optional. Can fix

Comment (zach2039) [reply src/main/java/dev/latvian/mods/tanky/client/TankControllerRenderer.java]: Corrected with [c981f35](https://github.com/latvian-dev/Tanky/pull/2/commits/c981f35a213c36d9fff25a18f3199993a5af39ac)

**Review by MaxNeedsSnacks — APPROVED** [review]

**Pull request merged**